packages/commonwealth/client/scripts/views/components/Comments/CommentEditor/CommentEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { CWText } from '../../component_kit/cw_text';
+import { User } from 'views/components/user/user';
+import Topic from 'client/scripts/models/Topic';
+import Account from 'client/scripts/models/Account';
+import app from 'state';
+import clsx from 'clsx';
+import { ReactQuillEditor } from '../../react_quill_editor';
+import { CWButton } from '../../component_kit/new_designs/cw_button';
+import { CWValidationText } from '../../component_kit/cw_validation_text';
+import BN from 'bn.js';
+import { weiToTokens, getDecimals } from 'helpers';
+import { ContentType } from 'types';
+import type { DeltaStatic } from 'quill';
+import './CommentEditor.scss';
+
+type CommentEditorProps = {
+  parentType: ContentType;
+  canComment: boolean;
+  handleSubmitComment: () => void;
+  errorMsg: string;
+  contentDelta: DeltaStatic;
+  setContentDelta: React.Dispatch<React.SetStateAction<DeltaStatic>>;
+  tokenPostingThreshold: BN;
+  activeTopic: Topic;
+  userBalance: BN;
+  disabled: boolean;
+  cancel: (e: any) => void;
```

Comment:
- `cancel => onCancel`
- i suggest we don't pass the react state callback in any component. prop update `setContentDelta => onContentDeltaChange` and in the callback in parent component we set local/react state
- `handleSubmitComment => onSubmitComment / onComment`

Reply:
`cancel` prop changed to `onCancel`.  `cancel` function changed to `handleCancel`.
I'm passing `setContentDelta` down to the Quill editor, which I'm not looking to modify.
I typically prefer using 'handle' for the functions and 'on' for props.